lib/ansible/modules/cloud/vmware/vmware_guest_file_operation.py

```diff
@@ -68,6 +68,12 @@
             - 'dns_name'
             - 'inventory_path'
             - 'vm_name'
+    vm_uuid_type:
```

Comment:
Doesn't it make much more sense to re-use the vm_id_type field here and just add `instance_uuid` (`uuid`) is already present. This way you can reuse that field (just make sure you update the code appropriately)

Otherwise: 
Missing `self.uuid_type = self.params['uuid_type']` in `__init__`